src/message_replies/timezone.js

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
This file can go, correct? Its not being used I dont think.

Reply:
yes. I thought that I had removed in my end but apparently not